.github/workflows/wikiman.yml

```diff
@@ -0,0 +1,21 @@
+name: wikiman
+
+on:
+  schedule:
+    - cron:  '6 12 * * *'
```

Comment:
This isn't gonna run before it's merged into main? Would be cool to see it run in a fork beforehand maybe?

Reply:
Or can we manually trigger it in the PR via workflow_dispatch?

Reply:
I don't believe we can trigger it manually until it is merged into the main branch

Reply:
In https://github.com/wbstack/mediawiki/pull/187/commits/0a6b26a3931d2f771e98c5b9e60c8afc1fbcc8ec I think I have made it so that this will run on PRs to this file too :)